obs-websocket-dotnet/OBSWebsocket.cs

```diff
@@ -320,6 +320,9 @@ public OBSWebsocket()
         /// <param name="password">Server password</param>
         public void Connect(string url, string password)
         {
+            if (url.ToLower().StartsWith("ws://") == false)
```

Comment:
Probably more consistant to modify it to:
`if (!url.To....` vs adding a `== false` at the end.